docs/09-status-melding.md

```diff
@@ -0,0 +1,154 @@
+Status feedback bepaald het succes van elk systeem en is er om de gebruiker te informeren over de status van het systeem. Er zijn drie verschillende vormen van status feedback (meldingen), te weten: <u>notificaties</u>, <u>validatoren</u> en <u>indicatoren</u>. Om gebruikers te kunnen helpen is het belangrijk te weten wanneer deze drie vormen van meldingen gebruikt worden en welke boodschap men daarbij wil aanduiden. De vorm van communiceren is afhankelijk van:
```

Comment:
`<u>` is een slecht idee, markdown biedt het niet voor niets niet aan. Ik adviseer schuin of bold, of een lijst werkvorm (`*`)